src/gaussian.jl

```diff
@@ -29,11 +29,6 @@ end
 
 pdf(P::Gaussian,x) = exp(logpdf(P::Gaussian, x))
 
-function Base.LinAlg.chol(u::UniformScaling{T}) where T<:Real
-    u.λ >= 0 || error(Base.LinAlg.PosDefException(1))
-    return UniformScaling{T}(sqrt(u.λ))
-end
-
 function Base.LinAlg.chol(u::SDiagonal{N,T}) where T<:Real where N
```

Comment:
No, not yet... and I thought I am done with these transfers. I created a pull request: https://github.com/JuliaArrays/StaticArrays.jl/pull/289